pkg/services/rendering/plugin_mode.go

```diff
@@ -9,10 +9,6 @@ import (
 )
 
 func (rs *RenderingService) renderViaPlugin(ctx context.Context, renderType RenderType, renderKey string, opts Opts) (*RenderResult, error) {
-	if renderType == RenderPDF {
-		opts.Encoding = "pdf"
-	}
```

Comment:
I prefer not to override a parameter and keeping this sync is hard. I'm using the `renderType` as the encoding value below.